Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Repository/ContentSubhypergraph.php

```diff
@@ -235,25 +255,22 @@ public function findParentNode(NodeAggregateId $childNodeAggregateId): ?Node
         ) : null;
     }
 
-    public function findNodeByPath(
-        NodePath $path,
-        NodeAggregateId $startingNodeAggregateId
-    ): ?Node {
+    public function findNodeByPath(NodePath $path, NodeAggregateId $startingNodeAggregateId): ?Node
+    {
         $currentNode = $this->findNodeById($startingNodeAggregateId);
```

Comment:
```suggestion
        $startingNode = $this->findNodeById($startingNodeAggregateId);
```

Oh yes, I'm gonna do all of it :D